course_inventory/inventory.py

```diff
@@ -64,7 +64,7 @@ def make_request_using_api_utils(url: str, params: Dict[str, Union[str, int]] =
     return response
 
 
-def slim_down_course_data(course_data: Sequence[Dict]) -> Sequence[Dict]:
+def slim_down_course_data(course_data: List[Dict]) -> List[Dict]:
```

Comment:
Why the change from `Sequence` → `List`?

Reply:
I was getting an error from `mypy` about argument types not matching, and this fixed it. I will recreate the error and post here.

Reply:
```course_inventory/inventory.py:113: error: Unsupported left operand type for + ("Sequence[Dict[Any, Any]]")```

Reply:
I guess where I'm using `+` between lists on 113, it doesn't like that the type hint is `Sequence`?

Reply:
From the readme. "Dict and List are aliases for the built-ins dict and list, respectively."

"Iterable, Sequence, and Mapping are generic types that correspond to Python protocols. For example, a str object or a List[str] object is valid when Iterable[str] or Sequence[str] is expected."

So it seems like the way Python duck typing works it will throw an error for these operands if it's not always possible to do the check. You can probably also check for isinstance(var, dict) and have mypy be good with it too later in the code. What you have here seems correct tho. Still learning this stuff.